server/gitrest/package.json

```diff
@@ -76,6 +76,7 @@
 	"packageManager": "pnpm@7.32.3+sha512.c28d73f0d82a6c4712a40b91aa43d7397e119b16f35e5e75fe9b7e8fd4f2fc6dfbb68bb3ffac3473a958bbafa7888b79ec384ad122537378089a3a2a19b9900e",
 	"pnpm": {
 		"overrides": {
+      "@types/node": "^16.18.16",
```

Comment:
Indented with spaces instead of tabs. Same for the other `server/**/package.json` files.

Reply:
How did prettier not catch these?

Reply:
#15855 should prevent this from happening again once merged. I cleaned up this PR already.